### ISSUE TITLE
Panel anchor target sizes are too small on mobile viewports

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#11411]
Initially reported by @albinazs as part of [WCAG 2.2 AAA* audit of the Wagtail admin – Nov 2023](https://github.com/wagtail/wagtail/discussions/11180).

In our collapsible sections, the anchor links on the _mobile_ viewport are too small (20x20).

This is failure of _2.5.8 Target Size (Minimum)_:

> The size of the target for pointer inputs is at least 24 by 24 CSS pixels, except when: Spacing, Equivalent, Inline, User Agent, Essential

Code in question:

```html
<a class="w-panel__anchor w-panel__anchor--suffix" href="#objects-in-workflow-section" aria-labelledby="objects-in-workflow-heading">
                <svg class="icon icon-link w-panel__icon" aria-hidden="true"><use href="#icon-link"></use></svg>
</a>
```

---

## Solution

Change the design of those buttons to be 24x24.

**Comment by Jaivignesh-afk** [comment]
Can i work on this issue?

**Comment by Jaivignesh-afk** [comment]
I believe I may have a potential solution for this problem. However, before proceeding, I would like to inquire about the necessity of implementing unit tests in this particular case. Your insights on this matter would be greatly appreciated.
